### ISSUE TITLE
Test Acc

### ISSUE THREAD

**Issue by suprola1017** [jinfagang/pytorch_image_classifier#4]
Why is the Train Loss decreasing continuously while the Test Loss remains unchanged or even increases when running this code? Is it due to overfitting?
